### PR TITLE
Add option to control pen types

### PR DESCRIPTION
**Pull request by soryy708**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -157,6 +157,30 @@
             <label for="filled-checkbox">Filled fields?</label>
             <i class="fa fa-question-circle has-tooltip" title="shows sample text in all [field]s"></i>
         </div>
+        <div class="input-field">
+            <label for="pen-style-select">Pen style</label>
+            <select id="pen-style-select">
+                <option value="Verdana, sans-serif" selected>Pen</option>
+                <option value="Comic Sans MS, sans-serif">Crayon</option>
+                <option value="Segoe Script, sans-serif">Fancy</option>
```
Comment (Dibasic) [inline index.html:170]: The fancy pen can only be #1c1713 and no other pens have that color (except a rainbow crayon set to exactly that hex, I suppose.)

If we don't want to handle populating that select just yet (and populating selectmenus in jQuery UI involves some frustrating issues, not certain glancing at this whether or not it ends up in a selectmenu by being in this dialog) then perhaps when 'fancy' is selected, the other select should automatically default to selecting the 'fancy' color.

**Comment by soryy708** [comment]
@Dibasic Changes done, ready for review.

**Review by Dibasic — CHANGES_REQUESTED** [review]
The fancy style-color binding doesn't seem to work and seems more complicated than it needs to be. Something like this:

```
    $('#pen-style-select').change(function () {
        if (this.val() === 'Fancy') {
            $('#pen-color-select').val('Fancy');
            updateFonts();
        }
    });
```
replacing the existing `.click()` binding should work, shouldn't it?

**Comment by soryy708** [comment]
@Dibasic What exactly doesn't seem to work for you?

**Comment by Dibasic** [comment]
> 
> 
> @Dibasic What exactly doesn't seem to work for you?

Selecting 'Fancy' as the pen style does not change the pen color to 'Fancy.'

**Comment by soryy708** [comment]
@Dibasic It does. Isn't fancy just a particular shade of black?

**Comment by soryy708** [comment]
image

**Comment by Dibasic** [comment]
If it does, it's not changing the selected option on the color select. That option should always match what the actual color is. I believe using `.val()` to change the selected value would make the displayed text also change.

**Comment by soryy708** [comment]
@Dibasic It does change on my machine. Which browser are you using?

**Comment by Dibasic** [comment]
Weird. I'm testing on Firefox 65.0.2.

**Comment by soryy708** [comment]
Odd. So am I!
Clear your cache?

Any errors in the dev console?

Did you, uhh, pull the latest version?

**Comment by Dibasic** [comment]
Tested again and working. Weird.

Looks good, just some merge conflicts to resolve and it's good to go.

**Review by Dibasic — APPROVED** [review]

**Comment by Dibasic** [comment]
Works. Looks nasty when used with pen colors, but that's for an issue.

**Pull request merged**